client/src/components/Logo.js

```diff
@@ -5,24 +5,24 @@ import styled from 'styled-components';
 import logo from '../styles/assets/logo.svg';
 
 const Wrapper = styled.a.attrs({
-    className: 'navbar-brand',
+  className: 'navbar-brand',
 })``;
 
 class Logo extends Component {
-    render() {
-        const { logoStyles } = this.props;
+  render() {
+    const { logoStyles } = this.props;
 
-        return (
-            <Wrapper href="https://localhost:8000">
-                <img src={logo} className="app--logo" style={logoStyles} alt="React Logo" />
-            </Wrapper>
-        );
-    }
+    return (
+      <Wrapper>
```

Comment:
If you remove the `href` here, maybe it should be in the styled component instead?
```js
const Wrapper = styled.a.attrs({
    className: 'navbar-brand',
    href: "/"
})``;
```